test/test_datapipe.py

```diff
@@ -42,6 +42,11 @@ def extract_datapipe_names(module):
     torchdata_datapipes = extract_datapipe_names(torchdata.datapipes.iter)
 
     missing_datapipes = pytorch_datapipes - torchdata_datapipes
+    deprecated_datapipes = {"FileLoader"}
```

Comment:
Adding this because `FileLoader` will stay in PyTorch Core for another release cycle (~3 months) before it is fully removed. Meanwhile, I don't think we should import it into TorchData (to discourage its usage).